internal/core/command/controller/http/command_test.go

```diff
@@ -349,10 +349,10 @@ func TestIssueGetCommand(t *testing.T) {
 
 	dic := NewMockDIC()
 	dic.Update(di.ServiceConstructorMap{
-		bootstrapContainer.MetadataDeviceClientName: func(get di.Get) interface{} { // add v2 API MetadataDeviceClient
+		bootstrapContainer.DeviceClientName: func(get di.Get) interface{} { // add v2 API DeviceClient
 			return dcMock
 		},
-		bootstrapContainer.MetadataDeviceServiceClientName: func(get di.Get) interface{} { // add v2 API MetadataDeviceProfileClient
+		bootstrapContainer.DeviceServiceClientName: func(get di.Get) interface{} { // add v2 API DeviceProfileClient
```

Comment:
mismatch between the code & comment on this line (comment should say `DeviceServiceClient`)

Reply:
Yep, and those comments are not useful ;-) Removed